staxing/assignment.py

```diff
@@ -737,7 +741,7 @@ def add_new_external(self, driver, title, description, periods,
             return
         self.select_status(driver, status)
 
-    def add_new_event(self, driver, title, description, periods, status,
+    def add_new_event(self, driver, title, description, periods, status, break_point=None):
```

Comment:
There's a continuation of the parameter list on the next line; by adding ` break_point=None):` to the same line, it's created a syntax error. I assume it exceeds the 79-character limit for the line. We should get the PEP-8 standards checker on your editor.